os.bpmsuite.businesscentral/added/launch/bpmsuite-businesscentral.sh

```diff
@@ -100,6 +100,8 @@ function configure_guvnor_settings() {
     JBOSS_BPMSUITE_ARGS="${JBOSS_BPMSUITE_ARGS} -Dorg.uberfire.metadata.index.dir=${bpmsuiteDataDir}"
     JBOSS_BPMSUITE_ARGS="${JBOSS_BPMSUITE_ARGS} -Dorg.uberfire.nio.git.dir=${bpmsuiteDataDir}"
     JBOSS_BPMSUITE_ARGS="${JBOSS_BPMSUITE_ARGS} -Dorg.uberfire.nio.git.ssh.cert.dir=${bpmsuiteDataDir}"
+    JBOSS_BPMSUITE_ARGS="${JBOSS_BPMSUITE_ARGS} -Dorg.uberfire.nio.git.daemon.enabled=false"
+    JBOSS_BPMSUITE_ARGS="${JBOSS_BPMSUITE_ARGS} -Dorg.uberfire.nio.git.ssh.host=0.0.0.0"
```

Comment:
I'm probably okay with 0.0.0.0, but are there any drawbacks we can think of? I know for smartrouter, for example, we do this instead: https://github.com/jboss-container-images/jboss-kie-modules/blob/master/os.bpmsuite.smartrouter/added/launch/bpmsuite-smartrouter.sh#L71-L87 . Thoughts?

Reply:
I have been able to clone/pull/push without any problem. If I am not wrong it must be bound to an existing IP in the container, the default is localhost. In the case you mention it will be used to connect to the router.

Reply:
Glad it works. I was just referencing the router case because I didn't know if we should similarly allow this to be configurable. However, I don't think 0.0.0.0 will hurt anything, so I'm okay with it.